samples/client/petstore/go/go-petstore-withXml/api/openapi.yaml

```diff
@@ -791,6 +791,7 @@ paths:
               properties:
                 integer:
                   description: None
+                  format: int32
```

Comment:
I don't think these changes are related to the PR?

Reply:
Oh I see you updated the samples, nevermind then!

Reply:
I guess either Swagger-Parser is setting those (explicit default), or Swagger-Core has a new behavior.

IMO this is an acceptable change.